__mocks__/mailchimp.mock.js

```diff
@@ -0,0 +1,39 @@
+const fs = require("fs");
+const path = require("path");
+
+const urlDirectoryMap = {
+  "api.mailchimp.com": "mailchimp"
+};
+
+const getData = url => {
+  let directory = "";
+  Object.keys(urlDirectoryMap).forEach(key => {
+    if (url.includes(key)) {
+      directory = urlDirectoryMap[key];
+    }
+  });
+  if (directory) {
+    const dataFile = fs.readFileSync(
+      path.resolve(__dirname, `./data/${directory}/response.json`)
+    );
+    const data = JSON.parse(dataFile);
+    return data[url];
+  }
+  return {};
+};
+
+const mailchimpGetRequestHandler = url => {
+    console.log("inside mailchimpget request handler");
+  const mockData = getData(url);
+  console.log(mockData)
+  if (!mockData) {
```

Comment:
If `!mockData` should we return 200? I think the check should be `if(mockdata)`

Reply:
Also the `data` should contain `mockData` as response, example `{ data: mockData, status: 200 }`